docs/codeql/codeql-language-guides/codeql-library-for-java.rst

```diff
@@ -307,8 +299,6 @@ For Javadoc, class ``Element`` has a member predicate ``getDoc`` that returns a
        jdoc = f.getDoc().getJavadoc()
    select jdoc
 
-➤ `See this in the query console on LGTM.com <https://lgtm.com/query/6022769142134600659/>`__. You can see this pattern in many projects.
```

Comment:
Is the "You can see this pattern in many projects." part worth keeping?

Reply:
Happy to keep it 👍🏻